app/src/main/java/com/example/mycleaninglog/MainActivity.kt

```diff
@@ -115,16 +139,6 @@ class MainActivity : ComponentActivity() {
 
         } else {
             Log.e("MainActivity.kt", "Error logging in " + response?.error?.errorCode)
-
         }
     }
-
-}
-
-@ExperimentalMaterialApi
-@Composable
-fun HomeScreen(myRooms: List<myRoom> = ArrayList<myRoom>()) {
-    //ExpandableCard(title = "Rooms", myRooms = myRooms, )
-    //ExpandableCard(title = "Common Tasks")
-    //ExpandableCard(title = "Upcoming Tasks")
```

Comment:
Good idea to remove commented out code.

Avoid pushing commented code to Version Control.  Version control has tools that makes this unnecessary.
- You can look at history to see what used to be there.
- You can use branches for experimental features.